contrib/cirrus/setup_environment.sh

```diff
@@ -54,6 +54,10 @@ if [[ "$CI_DESIRED_STORAGE" = "composefs" ]]; then
     if [[ "$PRIV_NAME" == "root" ]]; then
         CI_DESIRED_COMPOSEFS="+composefs"
 
+        # artifacts restored from a tarball won't be stored correctly
+        # in composefs, so we need to disable this feature.
+        export CI_NO_TEST_ARTIFACTS=true
```

Comment:
Why do we need another env var at all? If I wanted to run e2e locally now I need to set 

```
export STORAGE_FS=overlayfs
export STORAGE_OPTIONS_OVERLAY='overlay.use_composefs=true --pull-option=enable_partial_images=true --pull-option=convert_images=true'
````
and now yet another...
The e2e tests should be smart enough to know they are already configured to use composefs and then you can skip the cache. I don't see why anyone would want need this option without using composefs

Reply:
dropped the additional environment variable and just look if composefs is enabled